### PR TITLE
MF-890 - Add OPC-UA docs 

### LINKED ISSUE #890: Add OPC-UA docs

### PR DESCRIPTION
**Pull request by manuio**
Closes #890

### CODE REVIEW

File: docs/opcua.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+Bridging with an OPC-UA Server can be done over the [opcua-adapter](https://github.com/mainflux/mainflux/tree/master/opcua). This service sits between Mainflux and an [OPC-UA Server](https://en.wikipedia.org/wiki/OPC_Unified_Architecture) and just forwards the messages from one system to another.
+
+## Run OPC-UA Server
+
+The OPC-UA Server is used for connectivity layer. It allows various methods to read informations from the OPC-UA server and his nodes. The current version of the opcua-adapter still experimental and only `Read` and `Subscribe` methods are implemented.
```
Comment (drasko) [inline docs/opcua.md]: `his` is used for persons. For things you should use `its`.

File: docs/opcua.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
```
Comment (drasko) [inline docs/opcua.md:7]: Please mention public test-servers here.

**Review by drasko — CHANGES_REQUESTED** [review]

File: docs/opcua.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+### Route Map
+
+The opcua-adapter use [Redis](https://redis.io/) database to create a route-map between Mainflux and an OPC-UA Server. As Mainflux use Things and Channels IDs to sign messages, OPC-UA servers use Node Namespaces and Node Identifiers (the combination is called NodeID). The adapter route-mmap associate a `ThingID` with a `Node Identifier` and a `ChannelID` with a `Node Namespace`
+
+The opcua-adapter uses the matadata of provision events emitted by Mainflux system to update his route map. For that, you must provision Mainflux `Channels` and `Things` with an extra metadata key in the JSON Body of the HTTP request. It must be a JSON object with key `opcua` which value is another JSON object. This nested JSON object should contain `namespace` or `id` field. In this case `namespace` or `id` must be an existent OPC-UA `Node Namespace` or `Node Identifier`:
```
Comment (chombium) [inline docs/opcua.md]: use its instead of his as above

**Review by drasko — APPROVED** [review]
LGTM

**Review by nmarcetic — APPROVED** [review]

**Pull request merged**